### PR TITLE
RTC 386

### PR DESCRIPTION
**Pull request by Rados13**
Related to: https://github.com/jellyfish-dev/jellyfish/pull/104
Related to: https://github.com/jellyfish-dev/elixir_server_sdk/pull/37

After these changes, tests can be run only inside the docker. That's why a poetry command `ci_test` was added.
Tests can be run only inside docker because the jellyfish container can't access services running on the host OS through the `localhost` address.

### CODE REVIEW

File: tests/support/protos/jellyfish/__init__.py
Comment (roznawsk) [inline tests/support/protos/jellyfish/__init__.py:1]: I think we need this file, although the generated files are in two locations. The `tests/support/protos/jellyfish.py` should be removed. It should look like this:
image

File: jellyfish/events/_protos/protos/jellyfish/server_notifications_pb2.py
Comment (roznawsk) [inline jellyfish/events/_protos/protos/jellyfish/server_notifications_pb2.py:1]: We use python-betterproto for generating python code from definitions. 
You can run `./compile_proto.sh` script to generate it.
This file seems to be generated by the official proto compiler which I don't think is user friendly.

File: jellyfish/_notifier.py [hunk 1]
```diff
@@ -138,3 +138,16 @@ async def _subscribe_event(self, event: ServerMessageEventType):
...
+    def handle_json(json: Dict) -> Any:
+        '''
+        Transform received json notification to adequate notification instance.
+        '''
+        msg = json["notification"]
+        msg = bytes(msg, "utf-8")
+        message = ServerMessage().parse(msg)
+        _which, message = betterproto.which_one_of(message, 'content')
+        return message
+
```
Comment (roznawsk) [inline jellyfish/_notifier.py:153]: nitpick: add endline. I guess black would fix it in the next PR anyway

**Review by roznawsk — CHANGES_REQUESTED** [review]
Lots of good changes!
I like the improvements of poetry - adding the scripts for testing and checking lint and format.
I wonder if we should be testing every new feature in the SDK's - in this case check the new fields in the HLS component. But I guess we don't have to, since we are already testing it in Jellyfish.

**Review by Karolk99 — APPROVED** [review]
Looks good but as @roznawsk mentioned we have to agree on a specific test approach for our SDKs.

**Review by roznawsk — APPROVED** [review]

**Review by Karolk99 — APPROVED** [review]

**Review by roznawsk — APPROVED** [review]

**Pull request merged**